content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst

```diff
@@ -0,0 +1,153 @@
+not generate a transfer for the movement of finished products into stock. Inventory counts still
+update based on the number of products manufactured, but the act of transferring them to and from
+inventory is not tracked.
+
+.. tip::
+   The number of steps used in manufacturing is set at the warehouse level, allowing for each
+   warehouse to use a different number of steps. To change the number of steps used for a specific
+   warehouse, begin by navigating to :menuselection:`Inventory --> Configuration --> Warehouses`,
+   then select a warehouse from the :guilabel:`Warehouses` screen.
+
+   On the :guilabel:`Warehouse Configuration` tab, find the :guilabel:`Manufacture` radio input field
+   and select one of the three options: :guilabel:`Manufacture (1 step)`, :guilabel:`Pick components
+   and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then store
+   products (3 steps)`.
+
+   .. image:: two_step_manufacturing/manufacturing-type.png
+      :align: center
+      :alt: The Manufacture radio input field on a warehouse configuration page.
+
+.. important::
+   Products must be properly configured before they can be manufactured in Odoo. For details on how
+   to do so, see the documentation on how to :ref:`configure a product for manufacturing
+   <manufacturing/management/configure-manufacturing-product>`.
+
+Create manufacturing order
+==========================
+
+To manufacture a product in Odoo *Manufacturing*, begin by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, then click :guilabel:`New`
+to create a new manufacturing order.
+
+On the new manufacturing order, select the product to be produced from the :guilabel:`Product`
+drop-down menu. The :guilabel:`Bill of Material` field auto-populates with the associated Bill of
+Materials (BoM).
+
+If a product has more than one |BOM| configured for it, the specific |BOM| can be selected in the
+:guilabel:`Bill of Material` field, and the :guilabel:`Product` field auto-populates with the
+associated product.
+
+After a |BOM| has been selected, the :guilabel:`Components` and :guilabel:`Work Orders` tabs
+auto-populate with the components and operations specified on the |BOM|. If additional components or
+operations are required for the manufacturing order being configured, add them to the
+:guilabel:`Components` and :guilabel:`Work Orders` tabs by clicking :guilabel:`Add a line`.
+
+Finally, click :guilabel:`Confirm` to confirm the manufacturing order.
+
+Process pick components transfer
+================================
+
+After confirming a two-step manufacturing order, a :guilabel:`Transfers` smart button appears at the
+top of the page. Click it to open the pick components transfer for the manufacturing order. This
+transfer is used to track the movement of components from the locations where they are stored to the
+location where they are used to manufacture the product.
```

Comment:
```suggestion
transfer is used to track the movement of components from storage locations to manufacturing
locations for product assembly.
```
To just shorten this paragraph even more. Completely optional if brevity took away its original meaning (: